README.md

```diff
@@ -10,208 +10,124 @@
-    </thead>
-    <tbody>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/auto-device-selection.html">Auto</a></td>
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Auto plugin enables selecting Intel device for inference automatically</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/automatic-batching.html">Auto Batch</a></td>
-            <td><b><i><a href="./src/plugins/auto_batch">openvino_auto_batch_plugin</a></i></b></td>
-            <td>Auto batch plugin performs on-the-fly automatic batching (i.e. grouping inference requests together) to improve device utilization, with no programming effort from the user</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/hetero-execution.html">Hetero</a></td>
-            <td><b><i><a href="./src/plugins/hetero">openvino_hetero_plugin</a></i></b></td>
-            <td>Heterogeneous execution enables automatic inference splitting between several devices</td>
-        </tr>
-        <tr>
-            <td><a href="https://docs.openvino.ai/2024/openvino-workflow/running-inference/inference-devices-and-modes/multi-device.html">Multi</a></td>
-            <td><b><i><a href="./src/plugins/auto">openvino_auto_plugin</a></i></b></td>
-            <td>Multi plugin enables simultaneous inference of the same model on several devices in parallel</td>
-        </tr>
-    </tbody>
-</table>
+Welcome to OpenVINO™, an open-source software toolkit for optimizing and deploying deep learning models.
 
-## License
-OpenVINO™ Toolkit is licensed under [Apache License Version 2.0](LICENSE).
-By contributing to the project, you agree to the license and copyright terms therein and release your contribution under these terms.
+- **Inference Optimization**: Boost deep learning performance in computer vision, automatic speech recognition, natural language processing, and many other common tasks.
+- **Flexible Model Support**: Use models trained with popular frameworks such as TensorFlow, PyTorch, ONNX, Keras, and PaddlePaddle.
+- **Broad Platform Compatibility**: Reduce resource demands and efficiently deploy on a range of platforms from edge to cloud.
+- **Community and Ecosystem**: Join an active community contributing to the enhancement of deep learning performance across various domains.
 
-## Telemetry
-OpenVINO™ collects software performance and usage data for the purpose of improving OpenVINO™ tools. This data is collected directly by OpenVINO™ or through the use of Google Analytics 4.
-You can opt-out at any time by running the command:
+Check out the [OpenVINO Cheat Sheet](https://docs.openvino.ai/2024/_static/download/OpenVINO_Quick_Start_Guide.pdf) for a quick reference.
```

Comment:
let us use links with `latest` like `https://docs.openvino.ai/latest/_static/download/OpenVINO_Quick_Start_Guide.pdf`. Otherwise, we will have to update is again and again.
The same comment relates other links

Reply:
ok for now, but `latest` is definetely better